source/upgrade.txt

```diff
@@ -0,0 +1,98 @@
+.. _c-upgrade:
+
+=======================
+Upgrade Driver Versions
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: compatibility, backwards compatibility
+
+Overview
+--------
+
+This page describes the changes you must make to your application
+when you upgrade to a new version of the {+driver-short+}.
+
+Before you upgrade, perform the following actions:
+
+.. TODO: Link to platform support page
```

Comment:
[note for reviewer] Wasn't sure where this page would live in the updated docs structure so left as a todo for now.